infrastructure/variables.tf

```diff
@@ -44,3 +44,8 @@ variable "deployment_namespace" {
 variable "common_tags" {
   type = "map"
 }
+
+variable "evidence_upload_enabled" {
+  type = "string"
+  default = "false"
```

Comment:
Can we do it `default = ""`? As the config has a boolean here. Otherwise I change the flag to check for a string.

Reply:
Yes, otherwise `'false'` will turn the feature on

Reply:
Its a terraform variable type and it should map correctly boolean in your code.